Articles/Blog/2021-12-HPCAndScienceProcessCulture.md

```diff
@@ -77,7 +77,7 @@ What might a "mature" computational science center look like? I think the ASCI-c
 
 The Ops Manager's charge is to safeguard the scientific mission of the center by protecting its scientific output from being lost. The OM has budget, staff, and authority as required to carry out any duties required by this charge. The OM sets up collaboration modalities, chooses tools (including any lab notebook equivalents), and writes documentation. The OM also often attends meetings of the various working groups, and is literate in all aspects of the center's activities. The OM talks to everyone, formally and informally, and is alive to potential problems, as well as to opportunities to streamline operations and make collaborations easier. The OM is skilled at choosing solutions that solve problems, rather than hunting for problems that fit favored solutions. The OM sets expectations for how the technical staff deal with computations and analyses, and has authority to enforce those expectations. These expectations are no more burdensome than lab notebook protocols in chemistry labs: a properly-chosen OM performs his or her duties out in such a way  that the staff actually find it *easier* to work according to the OM-chosen rules than not to do so. Obviously, it is well-worth a Center Director's time to choose the OM very carefully, and hang on to a good one at any cost. A good OM is a defining feature of a research center in a mature field of computational science, so much so that proposal budgets for such centers are *required* to specify the OM role as a personnel line item, and to specify funds for OM operations commensurate with the duties of that role.
 
-So I guess the question that I've raised in this essay is: "Is Scientific HPC A Mature Field Yet?" Well, Scientific HPC is about 25, at least dating from the ASC Alliance program, and that's about enough time to make a grown-up.  Most of the first generation of seniors, together with their ~~bad management~~ imperfect habits should have retired by now, and it's been long enough that I'm guessing plenty of younger people have been noticing how much could be gained by raising our game at the management level. We can't count on Moore's law to make up in new, faster simulations for the mislaid, corrupted, or uninterpretable data from older ones, and that fact should be giving night sweats to quite a few people who write proposals for supercomputer time allocations. So I actually think I'm going to meet Geoff again quite soon. I still owe him a beer from the time I...um, well, that's a long story, and come to think of it he probably never realized it was me...  
```

Comment:
Just curios surrounding text in `~~` is the [GFM way](https://github.github.com/gfm/#strikethrough-extension-). Did that not work on rendered site?